modules/govuk_jenkins/templates/jobs/deploy_smokey_downstream.yaml.erb

```diff
@@ -12,7 +12,7 @@
           # Deploy to downstream environment
           CRUMB=$(curl https://<%= @jenkins_downstream_api_user %>:<%= @jenkins_downstream_api_password %>@<%= @deploy_url %>/crumbIssuer/api/json | jq --raw-output '. | .crumb')
 
-          curl -f -H "Jenkins-Crumb:$CRUMB" -XPOST https://<%= @jenkins_downstream_api_user %>:<%= @jenkins_downstream_api_password %>@<%= @deploy_url %>/job/Smokey_Deploy/build -d token=<%= @puppet_auth_token %>
```

Comment:
This is a straight revert of #11792, apart from [we're keeping the removal of the JSON variable](https://github.com/alphagov/govuk-puppet/pull/11792/files#diff-825e79bea45602dd599d2ba3ab655b4aec50aae15f555f46735ac6754ca46299L12)... right?

Reply:
Yes, we don't actually need either of those. I've added a new commit now